.github/workflows/cross-version-testing.md

```diff
@@ -0,0 +1,150 @@
+| :------------ | :----- | -------------------------------------------------- |
+| 0.20.3        | ✅     | The value of `minimum`                             |
+| 0.20.4        | ✅     | The latest micro version of `0.20`                 |
+| 0.21rc2       |        |                                                    |
+| 0.21.0        |        |                                                    |
+| 0.21.1        |        |                                                    |
+| 0.21.2        | ✅     | The latest micro version of `0.21` without`0.21.3` |
+| 0.21.3        |        | Excluded by `unsupported`                          |
+| 0.22rc2.post1 |        |                                                    |
+| 0.22rc3       |        |                                                    |
+| 0.22          |        |                                                    |
+| 0.22.1        |        |                                                    |
+| 0.22.2        |        |                                                    |
+| 0.22.2.post1  | ✅     | The latest micro version of `0.22`                 |
+| 0.23.0rc1     |        |                                                    |
+| 0.23.0        |        |                                                    |
+| 0.23.1        |        |                                                    |
+| 0.23.2        | ✅     | The latest micro version of `0.23`                 |
+| 0.24.dev0     |        |                                                    |
+| 0.24.0rc1     |        |                                                    |
+| 0.24.0        |        |                                                    |
+| 0.24.1        |        |                                                    |
+| 0.24.2        | ✅     | The latest micro version of `0.24`                 |
+| 1.0rc1        |        |                                                    |
+| 1.0rc2        |        |                                                    |
+| 1.0           |        | The value of `maximum`                             |
+| 1.0.1         | ✅     | The latest micro version of `1.0`                  |
+| 1.1.dev       | ✅     | The version installed by `install_dev`             |
+
+## When do we run cross version tests?
+
+1. Daily at 7:00 UTC using a cron scheduler.
+   [README on the repository root](../../README.rst) has a badge ([![badge-img][]][badge-target]) that indicates the status of the most recent run.
+2. When a PR that affects the ML integrations is created. Note we only run tests relevant to
+   the affected ML integrations. For example, a PR that affects files in `mlflow/sklearn` triggers
+   cross version tests for `sklearn`.
+
+[badge-img]: https://github.com/mlflow/mlflow/workflows/Cross%20version%20tests/badge.svg?event=schedule
+[badge-target]: https://github.com/mlflow/mlflow/actions?query=workflow%3ACross%2Bversion%2Btests+event%3Aschedule
+
+## How to run dev tests on a pull request
```

Comment:
This is the first occurrence of the term "dev tests". I think it would be nice to clarify what this refers to. I guess this is the same as cross-version tests because step 3 says "re-run the `cross-version-tests` workflow".

Reply:
```suggestion
## How to run cross version tests for dev versions on a pull request
```

I agree renaming `dev tests`. How about `cross version tests for dev versions`?

Reply:
So this implies that cross-version tests for dev versions are not enabled on a pull request by default. That makes sense. In addition to the renaming, maybe we can also add `Cross-version tests for dev versions are not enabled on a pull request by default.` to L127.